symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/model/BdkConfig.java

```diff
@@ -17,4 +17,7 @@
     private BdkExtAppConfig app = new BdkExtAppConfig();
     private BdkSslConfig ssl = new BdkSslConfig();
 
+    private BdkRetryConfig retry = new BdkRetryConfig();
```

Comment:
is the retry configuration something specific to DF API or should it apply to all clients?

Reply:
this is global retry config as I answered your comment about the retry configuration in DF config